gulpfile.js

```diff
@@ -201,22 +213,45 @@ gulp.task('vendor-scripts', () => beginPipe('./static/vendor/**/*.js')
 	.pipe(uglify())
 	.pipe(gulp.dest(`./build/${themeName()}/vendor`)));
 
+
+// =========================
+// COPY (Symlink) Files
+// =========================
+
 // copy other vendor files
-gulp.task('vendor-assets', () => beginPipe([
+gulp.task('vendor-assets', () => beginPipeLink([
```

Comment:
is this linking all file or just the folders which should be enough

Reply:
Good point. It currently links all files individually. I will adjust this :)

Reply:
Okay, I can't figure out how to link the whole folder at once. As far as I understand gulp now, it only reads files into the pipeline. Therefore we can't symlink folders, because they simply can't get read into the task.